Cargo.toml

```diff
@@ -2,7 +2,8 @@
 members = [
     "zingolib",
     "zingocli",
-    "zingoconfig"
+    "zingoconfig",
+    "zingo-memo-utils"
```

Comment:
Shall we rename this to "zingomemo"?

Reply:
Once we make that name change I am +1 on landing this.

Reply:
"zingomemo" matches the format of our other crates but i think "zingo-memo" is more readable which is my preference